usagov/usagov-2021
# Usagov 1089 remove extra footer space

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1089

## Description

There is extra space around the parentheses in the contact center phone number - but only shows in mobile. 
## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


1. Visit any page on dev and view the page in mobile
2. Look at the number in the footer to see if padding is removed from around the number in parenthesis
### Requires New Config
<img width="1512" alt="Screenshot 2023-10-11 at 1 21 05 PM" src="https://github.com/usagov/usagov-2021/assets/5551880/46e1bc60-619a-46fd-b353-09c2318642cc">

### Requires New Content

### Validation Steps
- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
